lxd/container_lxc.go

```diff
@@ -1264,6 +1273,44 @@ func (c *containerLXC) startCommon() (string, error) {
 					return "", err
 				}
 			}
+		} else if m["type"] == "nic" {
+			if m["nictype"] == "bridged" && shared.IsTrue(m["security.mac_filtering"]) {
```

Comment:
(same goes for ipv4.address, ipv6.address, limits.ingress, limits.egress, limits.max, ...)